File: Java/level-600.java
public class Main {
    private static final int FIRST_NUMBER = 1;
    private static final int LAST_NUMBER = 100;

    public static void main(String[] args) {
        for (int i = FIRST_NUMBER; i <= LAST_NUMBER; i++) {
            System.out.println(toFizzBuzzString(i));
        }
    }

    private static String toFizzBuzzString(int number) {
        String value = isFizz(number) ? "Fizz" : "";
        value += isBuzz(number) ? "Buzz" : "";
        return value.isEmpty() ? String.valueOf(number) : value;
    }

    private static boolean isFizzBuzz(int number) {
        return isFizz(number) && isBuzz(number);
    }

    private static boolean isFizz(int number) {
        return number % 3 == 0;
    }

    private static boolean isBuzz(int number) {
        return number % 5 == 0;
    }
}

File: PHP/level-600.php
<?php
    const FIRST_NUMBER = 1;
    const LAST_NUMBER = 100;

    for($i = FIRST_NUMBER; $i <= LAST_NUMBER; $i++) {
        echo toFizzBuzzString($i), "\n";
    }

    function toFizzBuzzString($number) {
      $value = isFizz($number) ? 'Fizz' : '';
      $value .= isBuzz($number) ? 'Buzz' : '';
      return empty($value) ? (string) $number : $value;
    }

    function isFizzBuzz($number) {
        return isFizz($number) && isBuzz($number);
    }

    function isFizz($number) {
        return $number % 3 == 0;
    }

    function isBuzz($number) {
        return $number % 5 == 0;
    }
?>
